pkg/asset/machines/powervs/machinesets.go

```diff
@@ -0,0 +1,95 @@
+// Package powervs generates Machine objects for IBM Power VS.
+package powervs
+
+import (
+	"fmt"
+
+	"github.com/openshift/installer/pkg/types/powervs"
+
+	machineapi "github.com/openshift/api/machine/v1beta1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// MachineSets returns a list of machinesets for a machinepool.
+func MachineSets(clusterID string, config *types.InstallConfig, pool *types.MachinePool, role, userDataSecret string) ([]*machineapi.MachineSet, error) {
+	if configPlatform := config.Platform.Name(); configPlatform != powervs.Name {
+		return nil, fmt.Errorf("non-powerVS configuration: %q", configPlatform)
+	}
+	if poolPlatform := pool.Platform.Name(); poolPlatform != powervs.Name {
+		return nil, fmt.Errorf("non-powerVS machine-pool: %q", poolPlatform)
+	}
+
+	var (
+		image, network string
+	)
+
+	platform := config.Platform.PowerVS
+	mpool := pool.Platform.PowerVS
+
+	if platform.ClusterOSImage != "" {
+		image = platform.ClusterOSImage
+	} else {
+		image = fmt.Sprintf("rhcos-%s", clusterID)
+	}
+	if platform.PVSNetworkName != "" {
+		network = platform.PVSNetworkName
+	} else {
+		network = fmt.Sprintf("pvs-net-%s", clusterID)
+	}
```

Comment:
Need this change here too
```suggestion
	image = fmt.Sprintf("rhcos-%s", clusterID)
	network = fmt.Sprintf("pvs-net-%s", clusterID)
	if platform.ClusterOSImage != "" {
		image = platform.ClusterOSImage
	}
	if platform.PVSNetworkName != "" {
		network = platform.PVSNetworkName
	}
```